pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -508,21 +508,19 @@ pub mod pallet {
 			domain_address: DomainAddress,
 			message: T::Message,
 		) -> (DispatchResult, Weight) {
-			let weight = Weight::from_parts(0, T::Message::max_encoded_len() as u64)
-				.saturating_add(LP_DEFENSIVE_WEIGHT);
```

Comment:
The defensive weight shadows the length of a message PoV. Anyways the `max_encoded_len` is not used here in any storage. Just process the content of the message, which is not dependent of the bytes of the message